kew.js

```diff
@@ -38,6 +38,40 @@
 			for (var i = 0; i < this.length; i++) {
 				this.elements[i].parentElement.removeChild(this.elements[i]);
 			}
+		},
+		add: function(htmlString) {
+			if (htmlString == "") {
+				throw new Error("Parameter in .add must not be an empty string.");
+			}
+
+			for (var i = 0; i < this.length; i++) {
```

Comment:
By using the reusing the same variable `i` inside your nested loop, you are essentially creating a **memory leak**.

Try using `kew.js` in a browser window - if you use the `add()` method, your tab gets frozen and keeps executing code until you close it. My tab went up to 5.8GB memory used before I had to kill it using Chrome's task manager.